src/MovieSection/MovieSection.scss

```diff
@@ -9,19 +9,40 @@
     // background-position: 50px 170em;
     // opacity: 0.9;
     background-repeat: no-repeat;
+    padding: 2%;
+
+    .user-welcome{
```

Comment:
I think for CSS style guide we want a space between the class property and opening curly brace
```
.user-welcome {
```